packages/styles/scss/components/masthead/_masthead-leftnav.scss

```diff
@@ -53,6 +53,13 @@
     }
   }
 
+  .#{$prefix}--side-nav {
+    &__header-navigation {
```

Comment:
Another thing I'm wondering is if there are any ways that we can avoid BEM concatenation by `&`. Though doing so goes out of our existing pattern of our Sass code, it helps sharing React style and Web Components style. Thanks!